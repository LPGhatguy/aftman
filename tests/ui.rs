use test_util::Environment;

#[test]
fn self_install() {
    let env = Environment::new(env!("CARGO_BIN_EXE_aftman"));

    let output = env.run(&["self-install"]);

    insta::assert_snapshot!("self-install stdout", output.stdout);
    insta::assert_snapshot!("self-install stderr", output.stderr);
}
