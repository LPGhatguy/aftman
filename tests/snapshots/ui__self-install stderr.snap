---
source: tests/ui.rs
assertion_line: 10
expression: output.stderr
---
[INFO  aftman::tool_storage] Updating all Aftman binaries...
[INFO  aftman::tool_storage] Installing Aftman...
[INFO  aftman::tool_storage] Updated Aftman binaries successfully!

